addons/website/__init__.py

```diff
@@ -37,8 +37,3 @@ def rem_website_id_null(dbname):
             ]).unlink()
 
     cr.postcommit.add(partial(rem_website_id_null, cr.dbname))
-
-
-def post_init_hook(cr, registry):
-    env = api.Environment(cr, SUPERUSER_ID, {})
-    env['ir.module.module'].update_theme_images()
```

Comment:
Hmmmm... interresting...
Given this is a method of the `website` module itself, and not a model method, I believe that a customization would have its own commit hook inside its own module... but indeed, given that the method did exist, maybe it could be called from a custom module. I will just make it `pass` instead... (and bad luck for a customization that would have called it in order to get `update_theme_images` executed)